### ISSUE TITLE
[T8][Accessibility][Functional]Unable to get focus on the Checkbox 

### ISSUE THREAD

**Issue by nehabug** [Mohanme13/Feb-19-Bug-bash#273]
'Past Medical' Tab - Can't focus on the Freetext checkbox beside Conditions.
